Assets/Scripts/Controllers/JobSpriteController.cs

```diff
@@ -22,13 +22,10 @@ public class JobSpriteController : MonoBehaviour
     // Use this for initialization
     void Start()
     {
-        jobGameObjectMap = new Dictionary<Job, GameObject>();
-        fsc = GameObject.FindObjectOfType<FurnitureSpriteController>();
-<<<<<<< HEAD
+		jobGameObjectMap = new Dictionary<Job, GameObject>();
+		fsc = GameObject.FindObjectOfType<FurnitureSpriteController>();
 
-=======
->>>>>>> FuelingSystem
-        WorldController.Instance.world.jobQueue.cbJobCreated += OnJobCreated;
+		WorldController.Instance.world.jobQueue.cbJobCreated += OnJobCreated;
```

Comment:
In this file it seems you went the other way around, spaces became tabs. :)

I think your Monodevelop isnt configured correctly. Check the contribution guidelines, I think there are instructions in there.